content/docs/dvclive/how-to/outputs.md

```diff
@@ -0,0 +1 @@
+# Track DVCLive outputs
```

Comment:
What's the plan for this page?

Reply:
Seems like maybe moving https://dvc.org/doc/dvclive/dvclive-with-dvc#outputs into here is still missing?

Reply:
Also not sure about term outputs here though.